sysutils/edid-decode/Portfile

```diff
@@ -0,0 +1,17 @@
+PortSystem       1.0
+
+name             edid-decode
+version          git-e48fb38
+categories       sysutils
+platforms        darwin
+maintainers      @wwalexander
+license          MIT
+description      Decode EDID data in human-readable format
+long_description edid-decode decodes EDID monitor description data in human-readable format
+homepage         https://git.linuxtv.org/edid-decode.git
+master_sites     git://linuxtv.org/edid-decode.git
```

Comment:
Since you are using `fetch.type git`, `master_sites` does not need to be set and can be omitted.

Reply:
Fixed (d73a890).